src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -249,7 +249,7 @@ bool WriteLog<I>::initialize_pool(Context *on_finish,
   }
   return true;
 
-error_handle:
+err_close_pool:
```

Comment:
Nit: in this case a bdev is closed, not a pool.  Normally I try to name these roughly based on the name of the cleanup function or its argument:
```suggestion
err_close_bdev:
```